### PR TITLE
[CIRCLECI] Use preinstalled CUDA

### PR DESCRIPTION
**Pull request by Skylion007**
This should speed up the CircleCI build by quite a bit and increase its reliability.

### CODE REVIEW

**Review by mathfac — COMMENTED** [review]
Looks good, @Skylion007 let's rebase and merge.

**Comment by mathfac** [comment]
@Skylion007 ping

**Comment by Skylion007** [comment]
@mathfac Waiting for approval

**Review by mathfac — APPROVED** [review]

**Pull request merged**
